lecture/questionnaires.md

```diff
@@ -1 +1,177 @@
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+
+**2.2)** At this stage, how confident do you feel about *<topic_of_interest>*?
+
+| **no, not at all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |  
+
+
+#### After:
+
+Having heard about *<topic_of_interest>*, please indicate how well prepared do 
+you feel to take on the following part of the project.
+
+Use the following scale to answer: no, not at all (1) - yes, entirely (7)
+
+**2.3)** How well prepared do you feel to perform *<topic_of_interest>*?
+
+| **no, not at all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+
+**2.4)** How confident do you feel about *<topic_of_interest>*?
+
+| **no, not at all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+    
+Please take a moment to evaluate the following aspects of the
+training session/lecture:
+
+**2.5)** How well informed we're you with regard to the current topic? (i.e.,
+did you know what was expecting you?)
+
+| **no, not at all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+
+**2.6)** Did the session/lecture provide enough opportunities for you to
+actively participate?
+
+| **no, not at all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+
+**2.7)** How satisfied are you with the content of the training/lecture?
+
+| **no, not at all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+
+**2.8)** How satisfied are you with the didactic methods used during
+the training/lecture?
+
+| **no, not ar all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+
+**2.9)** What was the most challenging part of the training/lecture?
+Did something in particular helped you master these challenges? If you
+didn't experience any difficulties, please tell us what the most 
+helpful/insightful part of the training/lecture was for you (< 100 words):
+
```

Comment:
should 2.9) include the question if something was missing or should this be an another question? E.g. "Was there content missing from the lecture, aspects you thought would be included or help you understand *<topic_of_interest>*?

Reply:
Oh yes, sorry, got lost somewhere, I'll add it right away